src/app/data-portal/data-portal-details/data-portal-details.component.ts

```diff
@@ -90,18 +91,58 @@ export class DataPortalDetailsComponent implements OnInit, AfterViewInit {
     @ViewChild('metadataPaginator') metadataPaginator: MatPaginator;
     // @ts-ignore
     @ViewChild('metadataSort') metadataSort: MatSort;
-// @ts-ignore
+    // @ts-ignore
     @ViewChild('annotationPaginator') annotationPaginator: MatPaginator;
     // @ts-ignore
     @ViewChild('annotationSort') annotationSort: MatSort;
-// @ts-ignore
+    // @ts-ignore
     @ViewChild('assembliesPaginator') assembliesPaginator: MatPaginator;
     // @ts-ignore
     @ViewChild('assembliesSort') assembliesSort: MatSort;
-// @ts-ignore
+    // @ts-ignore
     @ViewChild('filesPaginator') filesPaginator: MatPaginator;
     // @ts-ignore
     @ViewChild('filesSort') filesSort: MatSort;
+    // @ts-ignore
+    specSymbiontsTotalCount;
+
+    // @ts-ignore
+    dataSourceSymbiontsAssembliesCount;
+    // @ts-ignore
+    dataSourceSymbiontsAssemblies;
+    displayedColumnsAssemblies = ['accession', 'assembly_name', 'description', 'version'];
+
+    dataSourceMetagenomesAssembliesCount: any;
+    dataSourceMetagenomesAssemblies: any;
+    dataSourceSymbiontsRecords: any;
+
+
+    // @ts-ignore
+    metagenomesRecordsTotalCount: number | undefined;
+    dataSourceMetagenomesRecords: any;
+
+    // @ts-ignore
+    @ViewChild('metagenomesRecordsPaginator') metPaginator: MatPaginator;
+    // @ts-ignore
+    @ViewChild('metagenomesRecordsSort') metSort: MatSort;
+
+    // @ts-ignore
+    @ViewChild('assembliesMetagenomesPaginator') assembliesMetPaginator: MatPaginator;
+    // @ts-ignore
+    @ViewChild('assembliesMetagenomesSort') assembliesMetSort: MatSort;
+
+    specDisplayedColumns = ['accession', 'organism', 'commonName', 'sex', 'organismPart', 'trackingSystem'];
+
+    // @ts-ignore
+    @ViewChild('relatedSymbiontsPaginator') symPaginator: MatPaginator;
+    // @ts-ignore
+    @ViewChild('assembliesSymbiontsPaginator') asSymPaginator: MatPaginator;
+
+    // @ts-ignore
+    @ViewChild('relatedSymbiontsSort') relatedSymbiontsSort: MatSort;
+    // @ts-ignore
+    @ViewChild('assembliesSymbiontsSort') assembliesSymbiontsSort: MatSort;
+
```

Comment:
@ts-ignore is not the best option here, consider this line instead:
@ViewChild('metadataPaginator') metadataPaginator: MatPaginator | undefined;